### PR TITLE
[Task] Dark Theme for Testimonials and 404 External Pages

### PR DESCRIPTION
**Pull request by ErickDoroy**
## Related Links

- Task Link: https://app.asana.com/0/1202252431396579/1202289494314474/f
- Route Link: http://localhost:3000/swipet-journeys/Therese and http://localhost:3000/s

## Description

- Create a dark mode this pages
     - Testimonial page
     - 404 page

## Notes

N/A

## Screenshots

<details>
<summary>404 page</summary>

dark-mode-404_page

</details>
<details>
<summary>Testimonial page</summary>

dark-mode-testimonial

<details>
<summary>Mobile</summary>

dark-mode-testimonial-mobile

</details>

</details>

### CODE REVIEW

**Comment by ume45876139** [comment]
# Working

<details>
<summary>Screenshot</summary>

スクリーンショット 2022-05-18 18 52 04

</details>

- [x] Icon back ground color is too dark

<details>
<summary>Screenshot</summary>

スクリーンショット 2022-05-18 18 52 33

</details>

- [x] Name text color is too dark

<details>
<summary>Screenshot</summary>

スクリーンショット 2022-05-18 18 54 23

</details>

- [x] Date text color is too dark

<details>
<summary>Screenshot</summary>

スクリーンショット 2022-05-18 18 55 59

</details>

- [x] Text color is too dark
    - Should be same as footer (color: rgb(216 120 46/1);

# Testing data
test device: Chrome
test time: 0.5h

**Review by ume45876139 — APPROVED** [review]
# Working
## Chrome
<details>
<summary>PC</summary>

https://user-images.githubusercontent.com/39412730/169194062-db59ff88-f65d-4370-9f40-df0b7132cae0.mp4

</details>

<details>
<summary>Tablet</summary>

https://user-images.githubusercontent.com/39412730/169194208-c6d700fb-b204-4692-b676-136232abcd5a.mp4

</details>

<details>
<summary>iPhone</summary>

https://user-images.githubusercontent.com/39412730/169194253-63daa039-df48-4b8c-acc4-cd1b355084ea.MP4

</details>

- Text color is proper
- Icon color is proper
- No deformation


# Testing data
test device: Chrome, Safari (iPhone), Firefox, Opera
test time: 0.5h
comment: 4
iteration: 1
total test time: 1h

File: src/pages/components/externalPages/TestimonialExternalPages/TestimonialContentTemplate.js [hunk 1]
```diff
@@ -2,18 +2,26 @@ import React, { Fragment } from 'react';
...
 import TestimonialMainContent from './TestimonialMainContent';
 
-const TestimonialContentTemplate = ({ description }) => (
+const TestimonialContentTemplate = ({ mode, description }) => (
   <Fragment>
-    <header className='flex px-[20px] pt-[64px] sm:px-[55px] sm:pt-[38px] lg:px-[100px] lg:pt-[51px] xl:px-[200px] 2xl:px-[339px]'>
-      <LazyLoadImage src='/images/Avatar01.svg' alt='profile' className='h-[71px]' />
+    <header
+      className={`${
+        mode === 'light' ? '' : 'bg-dark-ash-800'
```
Comment (egiegarciano14) [inline src/pages/components/externalPages/TestimonialExternalPages/TestimonialContentTemplate.js]:
```suggestion
        dark:bg-dark-ash-800
```
I think you make it like this

File: src/pages/components/externalPages/TestimonialExternalPages/TestimonialContentTemplate.js [hunk 2]
```diff
@@ -2,18 +2,26 @@ import React, { Fragment } from 'react';
...
-        <h1 className='text-[32px] text-navy-900'>{description.author}</h1>
-        <p>{description.posted}</p>
+        <h1 className='text-[32px] text-navy-900 dark:text-white '>{description.author}</h1>
+        <p className='dark:text-white'>{description.posted}</p>
       </div>
     </header>
     <main
-      className='border-brown-90 border-b px-[20px]
-          text-lg text-brown-900 sm:px-[55px] sm:pt-[38px] md:px-[50px] lg:px-[100px] lg:pt-[51px] xl:px-[200px] 2xl:px-[339px]'
+      className={`${mode === 'light' ? '' : 'bg-dark-ash-800'} px-[20px]
```
Comment (egiegarciano14) [inline src/pages/components/externalPages/TestimonialExternalPages/TestimonialContentTemplate.js]:
```suggestion
      dark:bg-dark-ash-800
```

File: src/pages/components/externalPages/TestimonialExternalPages/TestimonialHeaderContent.js [hunk 3]
```diff
@@ -2,51 +2,68 @@ import { Link } from 'react-router-dom';
 import { LazyLoadImage } from 'react-lazy-load-image-component';
 import DownloadButton from '../../DownloadButton';
 
-const TestimonialHeaderContent = ({ item }) => (
-  <figure className='relative h-[572px] pb-5 sm:h-[450px] md:h-[550] lg:h-[572px] xl:h-[595px]'>
+const TestimonialHeaderContent = ({ mode, item }) => (
+  <figure
+    className={`${
+      mode === 'light' ? '' : 'bg-dark-ash-800'
```
Comment (egiegarciano14) [inline src/pages/components/externalPages/TestimonialExternalPages/TestimonialHeaderContent.js]:
```suggestion
      dark:bg-dark-ash-800
```

**Review by egiegarciano14 — APPROVED** [review]

**Pull request merged**